download_and_create_reference_datasets/v02/mito/create_ht__gnomad_38.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+
+# Download gnomAD mito datset vcf file, import it to a matrix table, annotate required fields into al hail table,
+# and save it to a file.
+
+import logging
+import hail as hl
+
+logging.basicConfig(format='%(asctime)s %(levelname)-8s %(message)s')
+logger = logging.getLogger()
+
+CONFIG = {
+    'vcf_path': 'gs://gcp-public-data--gnomad/release/3.1/vcf/genomes/gnomad.genomes.v3.1.sites.chrM.vcf.bgz',
+    'output_path': 'gs://seqr-reference-data/GRCh38/mitochondrial/gnomad.genomes.v3.1.chrM.ht',
+}
+
+MITO_INFO_FIELDS = ['AN', 'AC_hom', 'AC_het', 'AF_hom', 'AF_het', 'max_hl']
+OUTPUT_FIELDS = ['locus', 'alleles', 'rsid'] + MITO_INFO_FIELDS
+
+
+def vcf_to_mt(path):
+    """
+    Converts vcf to mt.
+
+    :param path: vcf path
+    :return:
+    """
+    logger.info(f'Loading VCF file {path}')
+    mt = hl.import_vcf(path, reference_genome='GRCh38')
```

Comment:
for other reference datasets we use a `.ht` reference and call `hl.read_table`. Is there any pro/con for using `.mt` and `hl.import_vcf` here instead?

Reply:
I agree with using the ht. The gnomad  VCF and HT files are sites only so there are only rows with annotations, no entries or samples, so the mt assignment is confusing.

Reply:
Yes and its not a big deal but gnomAD releases the HT already so we could just use that instead of taking the time to read in and convert the VCF.

Reply:
If I understand it correctly, the per data source scripts in `download_and_create_reference_datasets/v02/hail_scripts` are for reading in the inputs and saving to per data source hail tables. The `write_combined_reference_data_ht.py` reads in the hail tables for each source and writes to a combined hail table.
We could do the same thing. Or we could change to create a single combination script and some configuration files, one configuration file for each data source.

Reply:
As I stated in my previous comment, we should model oursleves after `write_combined_reference_data_ht.py`. There is no advantage to maintaining many separate files for configuring and downloading tables, having a single file with all the configurations is by far the easiest to read and maintain